mod-invoice-storage/schemas/adjustment.json

```diff
@@ -3,26 +3,65 @@
   "description": "Adjustment",
   "type": "object",
   "properties": {
-    "adjustmentLabel": {
+    "id": {
+      "description": "UUID of this adjustment",
+      "type": "string",
+      "pattern": "^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[1-5][0-9a-fA-F]{3}-[89abAB][0-9a-fA-F]{3}-[0-9a-fA-F]{12}$"
+    },
+    "description": {
       "description": "Displayed in invoice line as a name for the input",
       "type": "string"
     },
-    "adjustmentType": {
+    "fundDistributions": {
+      "description": "Fund distribution records for this purchase order line",
+      "id": "fundDistributions",
+      "type": "array",
+      "items": {
+        "description": "fund distribution record",
+        "type": "object",
+        "$ref": "fund_distribution.json"
+      }
+    },
+    "prorate": {
+      "description": "Displayed in invoice line per adjustment in toggled on in settings",
+      "type": "string",
+      "enum": [
+        "By line",
+        "By amount",
+        "By quantity",
+        "Not prorated"
+      ],
+      "default" : "Not prorated"
+    },
+    "relationToTotal": {
+      "description": "Relationship of this adjustment to the total;In addition to: added to subtotal;Included in: reported as subtotal portion;Separate from:calculated from subtotal",
+      "type": "string",
+      "enum": [
+        "In addition to",
+        "Included in",
+        "Separate from"
```

Comment:
Don't we also need "Not prorated" in the enum list too?

Reply:
oh wait, I'm looking at the wrong field.. but I don't think we want "Not prorated" here as the default... i think we want "In addition to"